articles/iot-edge/how-to-deploy-cli-at-scale.md

```diff
@@ -162,6 +162,12 @@ The previous example showed a layered deployment setting the `properties.desired
 }
 ```
 
+In summary, to create a layered deployment:
+
+- Must add the `--layered` flag to the Azyre CLI create command
+- Must not contain any system modules, cam only user modules
+- Must use the 'dot notation' under `$edgeAgent` and `$edgeHub`
```

Comment:
>Must use the 'dot notation' under `$edgeAgent` and `$edgeHub` [](http://example.com/codeflow?start=2&length=61)

The example you gave earlier was very illustrative, do you think we should add it in the docs too - properties.desired.[base] $edgeAgent.modules: { version: 2}
[layered] properties.desired.$edgeAgent.modules.version: '2'

Reply:
That example was for the two different ways you could write in the layered deployment, both of which are valid. Do you think it would still be good to add it?